apps/summerfi-api/lib/sdk-router-function/src/procedures/providerProcedure.ts

```diff
@@ -1,10 +1,19 @@
+import { TokenAmount } from '@summerfi/sdk-common/common'
+import type { GetQuote } from '@summerfi/simulator-service'
 import { publicProcedure } from '~src/trpc'
 
 export const providerProcedure = publicProcedure.use(async (opts) => {
-  // TODO: Add reusable procedure logic here
+  const getQuoteMock: GetQuote = async ({ fromTokenAmount, fee, slippage, toToken }) => {
```

Comment:
I'm confused about this file: it is called `providerProcedure` but it implements a mock for getQuote. What is the intention?

Reply:
this was only temporary mock while it was not implemented